pyproject.toml

```diff
@@ -51,3 +51,58 @@ max-complexity = 8
 
 [tool.ruff.flake8-quotes]
 inline-quotes = "double"
+
+[tool.bumpversion]
+current_version = "0.5.0"
+commit = false
+tag = false
+parse = """
+(?P<major>\\d+)\\.(?P<minor>\\d+)\\.(?P<patch>\\d+)
+(?:
+(?P<prerelease>alpha|beta|rc)
+(?P<build>\\d+)
+)?
+$
+"""
+serialize = [
+	"{major}.{minor}.{patch}{prerelease}{build}",
+	"{major}.{minor}.{patch}",
+]
```

Comment:
We can get rid of the "parse" and "serialize" sections and just use the default behavior. We don't need to think about "alpha" or "beta" or "rc", at least not yet.

Reply:
And can get rid of `[tool.bumpversion.parts.prerelease]` and `[tool.bumpversion.parts.build]` sections